tasks/src/visual-question-answering/about.md

```diff
@@ -0,0 +1,43 @@
+## Use Cases
+
+### Automatic Medical Diagnostic
+Visual Question Answering (VQA) models can be used to help clinical staff diagnose conditions. For example, a doctor could input an MRI scan to the model and ask "what does this MRI scan show?" and get the answer to their question. 
+
+###  Aid the Visually Impaired Persons
+VQA models can be used to reduce visual barriers for visually impaired individuals by allowing them to get information about images from the web and the real world.
+
+### Unattended Surveillance
+Video Question Answering can be used to quickly extract valuable information from surveillance recordings. 
```

Comment:
For this one, I think a basic activity detection model would be enough and VQA is too overkill.